app/components/gleam-logo.ts

```diff
@@ -0,0 +1,130 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { Rive } from '@rive-app/canvas';
+
+interface GleamLogoSignature {
+  Element: HTMLDivElement;
+  Args: {
+    class?: string;
+    style?: string;
+    [key: string]: unknown;
+  };
+  Blocks: Record<string, never>;
+}
+
+export default class GleamLogoComponent extends Component<GleamLogoSignature> {
+  @tracked riveInstance: Rive | null = null;
+  container: HTMLElement | null = null;
+  animationInterval: number | null = null;
+
+  @action
+  cleanupRive() {
+    if (this.animationInterval) {
+      clearInterval(this.animationInterval);
+      this.animationInterval = null;
+    }
+
+    if (this.riveInstance) {
+      this.riveInstance.stop();
+      this.riveInstance = null;
+    }
+  }
+
+  @action
+  handleMouseEnter() {
+    if (this.riveInstance) {
+      const stateMachines = this.riveInstance.stateMachineNames;
+
+      if (stateMachines && stateMachines.length > 0) {
+        const stateMachineName = 'State Machine 1';
+
+        if (stateMachines.includes(stateMachineName)) {
+          this.riveInstance.reset();
+          this.riveInstance.play(stateMachineName);
+        }
+      }
+    }
+  }
+
+  @action
+  handleMouseLeave() {
+    if (this.riveInstance) {
+      this.riveInstance.stop();
+      this.riveInstance.reset();
+    }
+  }
+
+  @action
+  setupRive(element: HTMLDivElement) {
+    this.container = element;
+
+    try {
+      const canvas = document.createElement('canvas');
+
+      // Set the canvas size to be 4x the original size for better quality
+      canvas.width = 141 * 4;
+      canvas.height = 144 * 4;
+
+      // Scale the canvas to fit the container while maintaining aspect ratio
+      canvas.style.width = '100%';
+      canvas.style.height = '100%';
```

Comment:
@Arpan-206 let's confirm if this actually does preserve aspect ratio? According to the numbers above the file isn't exactly square shaped